internal/grep/grep_test.go

```
@@ -29,20 +33,39 @@ func foo() {}`), 0644)
 	for f, rem := range expected {
-		if rem {
-			t.Errorf("missing expected file: %s", f)
-		}
+		assert.False(t, rem, "missing expected file: %s", f)
 	}
 }
+
+func TestSearchFiles_ContextCancellation(t *testing.T) {
+	root := t.TempDir()
+	// ライブラリ用フォルダ
+	os.MkdirAll(filepath.Join(root, "pkg"), 0755)
```

golangci-lint (1.64.8)

Error return value of `os.MkdirAll` is not checked

(errcheck)